security/accesscontrol.rst

```diff
@@ -41,6 +41,8 @@ Authorization in Firely Server by default is based on `SMART on FHIR`_ and more
 * scope=[array of individual scopes]
 * patient=123: the user is allowed access to resources in the compartment of patient 123 -- see :ref:`feature_accesscontrol_compartment`.
 
+.. note:: To complete a write request (POST, PUT or DELETE) with a conditional statement (see https://hl7.org/fhir/http.html) requires read permission on the condition. |br| Therefore, user/\*.write will usually require additional ``read`` scopes.
```

Comment:
```suggestion
.. note:: A conditional create, update or delete (see the `FHIR http specification <https://hl7.org/fhir/http.html>`_), requires read permissions on the condition. |br| Therefore, ``user/*.write`` will usually require additional ``read`` scopes.
```
 I suggest to use 'conditional create ...', since that is the term used on the specification page that you link to.

Reply:
I committed your suggestion and removed the |br| because for some reason the build says 'D:\Firely\firely-docs-firely-server\security\accesscontrol.rst:44: ERROR: Undefined substitution referenced: "br".'